### DISCUSSION TITLE
Running HammerDB DockerFile-built container in any K8s distro is failing

### DISCUSSION THREAD

**Discussion by rfsonders** [TPC-Council/HammerDB#493] [Q&A] [answered]
Hi folks.
I am attempting, without success, to run the Dockerfile-built container, deployed to any K8s distro. Even testing with the small k8s docker desktop distro. The container loops with CrashLoopBackOff. Over and over. Also tested with AKS. Same error.

The container does run fine, directly in docker. I can bash right in and execute .tcl, not a problem.
I am at a loss as to why the same container will not start in k8s.

Any help, thoughts, or direction would be appreciated.
Thank you!

**Comment by pooja-jain-17** [comment]
Hi Robert, Thanks for trying this. I haven't tried with Kubernetes so thanks for the insight. I will look into it. -Pooja

**Comment by sm-shaw** [comment]
Opened issue https://github.com/TPC-Council/HammerDB/issues/495

**Comment by sm-shaw** [comment, marked as answer]
This question is answered by @pooja-jain-17 in the related issue here https://github.com/TPC-Council/HammerDB/issues/495#issuecomment-1414551715
